Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ruby:2.3.1
+RUN apt-get update -qq && apt-get upgrade -y && apt-get install -y build-essential nodejs && apt-get clean
```

Comment:
Does this need nodejs installed?

Reply:
The gemfile has `uglifier` as a dependency so requires a JS runtime